### ISSUE TITLE
How to CONTRIBUTE?

### ISSUE THREAD

**Issue by findel** [ShyykoSerhiy/skyweb#50]
Could you add a contribution / build document? 

I'd like to work on this using VS Code, but I'm having trouble getting it set up correctly to build/compile.

**Comment by ShyykoSerhiy** [comment]
Of course. I'm going to make it more "contributable" this Saturday along with added tests, contribution guide and some refactoring.

**Comment by findel** [comment]
Brill. I think I've got it working now.

It was not finding the `typings/index.d.ts` file.

I needed to `npm install typings -g` and then run `typings install debug --save`. 

Now it seems to be compiling to JavaScript in dist, but I get errors like these:

```
src/contacts_service.ts(1,26): error TS2307: Cannot find module 'request'.
src/contacts_service.ts(5,23): error TS2307: Cannot find module 'http'.
src/contacts_service.ts(6,25): error TS2307: Cannot find module 'request'.
```

Can I fix those?

**Comment by ShyykoSerhiy** [comment]
That's strange. 
All you need to do is

``` sh
git clone https://github.com/ShyykoSerhiy/skyweb.git
cd ./skyweb
npm i
npm i -g typings typescript
typings install
tsc 
```

There should be no errors.

**Comment by ShyykoSerhiy** [comment]
It's `typings install debug --save` line that's wrong. You need 'typings install'. It restores typings that are provided in typings.json file. `typings install debug --save` actuall install typings for debug package and adds it to typings.json file. Hope this helps.

**Comment by findel** [comment]
Yup, that's good. I've got it building okay now. Thanks.

**Comment by qvaqvaboo** [comment]
Reproduces for me:

```
git clone https://github.com/ShyykoSerhiy/skyweb.git
cd ./skyweb
npm i
npm i -g typings typescript
typings install
tsc 
```

Results in:
`error TS6053: File '/skyweb/typings/index.d.ts' not found.`

Had to do 
`typings install debug --save`
 first
